### PR TITLE
Fixes for #20 #21 

### PR DESCRIPTION
**Pull request by jacek-marchwicki**
Fixes for #20 #21 

CLA already signed
Build successed: https://registry.hub.docker.com/u/appunite/cloud-sdk/build_id/35948/code/bo4vtfvtpkdbxmyb3juz49/

Cheers :)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FROM google/debian:wheezy
 ENV DEBIAN_FRONTEND noninteractive
-RUN sed -i '1i deb     http://gce_debian_mirror.storage.googleapis.com/ wheezy         main' /etc/apt/sources.list
-RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client && apt-get clean
+RUN apt-get update && apt-get install -y -qq --no-install-recommends wget unzip python php5-mysql php5-cli php5-cgi openjdk-7-jre-headless openssh-client python-openssl && apt-get clean
```
Comment (proppy) [inline Dockerfile]: can you change apt-get clean,  with rm -rf /var/lib/apt/lists/*? I think google/debian already does apt-get clean after each install now.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,20 @@ Compute Engine:
        $ gcloud config set project <project>
     gcloud config set project ...
 
+    # If you would like to use service account instead please look here:
+    docker run --volume /.config/gcloud/ \
```
Comment (proppy) [inline README.md]: alternatively you could just name the `gcloud auth activate-service-account` container instead of using a data container.

I don't mind either way, but we should be consistent with the rest of the README file, i.e: if we switch to data container (maybe with tianon/true, instead of busybox), we should also change the example below.

**Comment by proppy** [comment]
Sorry for the late followup, this looks good to me and thanks for reporting *and* fixing the issue, just 2 little nits.

File: Dockerfile (hunk 1 above)
Comment (jacek-marchwicki) [inline Dockerfile]: This was not a point of my change. 
I think you can do this in separate changeset.

Comment (proppy) [inline Dockerfile]: Sure makes total sense ;)

**Comment by proppy** [comment]
Thanks again!
